Sources/Networking/Operations/PostReceiptDataOperation.swift

```diff
@@ -332,6 +337,13 @@ private extension EncodedAppleReceipt {
             return content.asData.hashString
         case let .receipt(data):
             return data.hashString
+        case let .sk2receipt(receipt):
+            do {
+                return try receipt.prettyPrintedData.hashString
```

Comment:
Switched to `prettyPrintedData` to generate consistent hashes on iOS 17+

Reply:
Oooh great catch